ndk-make.sh

```diff
@@ -1,60 +1,61 @@
 #!/bin/sh
+
+# If you want to speed up compilation, you can run this script with your
+# architecture as an argument:
+#
+#     ./ndk-make.sh arm64-v8a
+#
+# Possible values are armeabi-v7a, arm64-v8a, x86 and x86_64.
+# You should be able to find out your architecture by running:
+#
+#     adb shell uname -m
+#
+# or:
+#
+#     adb shell cat /proc/cpuinfo
+#
+# The values in the following lines mean the same:
+#
+# armeabi-v7a, armv7 and arm
+# arm64-v8a, aarch64 and arm64
+# x86 and i686
+# (there are no synonyms for x86_64)
+#
+#
+# If you put this in your .bashrc, then you can directly build and
+# deploy DeltaChat from the jni/deltachat-core-rust directory by
+# typing `nmake`:
+#
+# nmake() {(cd ../..; ./ndk-make.sh arm64-v8a && ./gradlew installFatDebug; notify-send "install finished")}
+
 set -e
 echo "starting time: `date`"
 
-cd jni/deltachat-core-rust
+: "${ANDROID_NDK_ROOT:=$ANDROID_NDK_HOME}"
+: "${ANDROID_NDK_ROOT:=$ANDROID_NDK}"
+if test ! -z "$ANDROID_NDK_ROOT"; then
+    echo Setting CARGO_TARGET environment variables.
 
-# to setup the toolchains (from https://medium.com/visly/rust-on-android-19f34a2fb43 )
-# run `scripts/install-toolchains.sh`.
-#
-# Currently ndk20b is minimum required version
-# (newer versions will likely work, however, are not tested and not used in offial releases,
-# in general, changes on the ndk-version should be done with care)
-#
-# after that, add PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin to your $PATH
-# and add the correct clang-linkers to `~/.cargo/config`:
-# ```
-# [target.armv7-linux-androideabi]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/armv7a-linux-androideabi16-clang"
-# [target.aarch64-linux-android]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/aarch64-linux-android21-clang"
-# [target.i686-linux-android]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/i686-linux-android16-clang"
-# [target.x86_64-linux-android]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/x86_64-linux-android21-clang"
-# ```
-# then, the following should work:
-# (If you want to speed up compilation, have look at the comment in ndk-make-fast.sh.)
+    if -z "$NDK_HOST_TAG"; then
```

Comment:
```suggestion
    if test -z "$NDK_HOST_TAG"; then
```

that works on mac, not sure, what `if -z` is for, maybe just a typo or so :)